src/math_parser.cpp

```diff
@@ -549,6 +549,10 @@ void math_exp::math_exp_impl::new_var( std::string_view str )
                 debugmsg( "Unknown scope %c in variable %.*s", str[0], str.size(), str.data() );
         }
     }
+    if( str.size() > 1 && str[0] == '_' ) {
```

Comment:
```suggestion
    } else if( str.size() > 1 && str[0] == '_' ) {
```
Otherwise variables like `u__blorg` get scoped twice.